### PR TITLE
feat : Added cart management

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Added a cart Page from where orders can be managed

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

**Review by fahd23 — APPROVED** [review]
Looks good to merge

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    font-size: 3rem;
+}
+
+.card-btn {
+    width: 4rem;
+    height: 1rem;
+}
+
+.ecom-quant {
+    width: 30px;
```
Comment (kotalwar-krishna) [inline Cart/cart.css:42]: be consistent using units. some times for width yu uses px and some time rem.

File: Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.quant-count {
+    text-align: center;
+}
+
+.btn-change {
+    width: 2rem;
```
Comment (kotalwar-krishna) [inline Cart/cart.css:50]: refor for above comment

**Review by kotalwar-krishna — APPROVED** [review]
Other things look good please refer saggestions

File: Cart/cart.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Cart/cart.css:42]: Yeah, here I have used pixels as I dont want to change the width when the screen is resized or viewed on different devices. Anyway, Thanks for the suggestion. I will be consistent using units for lengths

**Pull request merged**